src/tests/data/testData.ts

```diff
@@ -0,0 +1,23 @@
+import { EquipmentPiece, Player } from '@/types/Player';
+import { PartialDeep } from 'type-fest';
+import equipment from '../../../cdn/json/equipment.json';
+
+const availableEquipment = equipment.map((v) => v as EquipmentPiece);
+
+const findItemByName = (name: string): EquipmentPiece | undefined => availableEquipment.find((eq) => eq.name === name);
```

Comment:
I know in the EquipmentPresets.tsx we used item ids instead of name, but it seems like it could become a little cumbersome especially if we have a tone of different gear sets for tests.

Reply:
I think it'd be useful to keep using IDs here aswell, as it ensures that there is consistency between what code is running in the app and what the tests are running.

To make things easier, perhaps the switch case in the equipment preset select component can be moved to a utility function somewhere else, and called from both here and that component. That way, the test is always using the exact same code as the app, and we increase our code coverage.